packages/functional-tests/tests/settings/totp.spec.ts

```diff
@@ -53,6 +58,26 @@ test.describe('severity-1 #smoke', () => {
     expect(status).toEqual('Enabled');
   });
 
+  test('add TOTP and confirm sync signin', async ({
+    credentials,
+    target,
+    pages: { login, settings, totp, page, connectAnotherDevice },
+  }) => {
+    await settings.goto();
+    await settings.totp.clickAdd();
+    await totp.enable(credentials);
+    await settings.signOut();
+
+    // Sync sign in
+    await page.goto(
```

Comment:
I think this could cause some flaky tests, you might need to load the [pages](https://github.com/mozilla/fxa/blob/2fcfd881cb130eff6b1ee2228ad161aa408b8dd9/packages/functional-tests/lib/fixtures/standard.ts#L121).

A cleaner fix is to probably patch FF and not show the account merge dialog.

Reply:
Thanks @vbudhram for the review, I will make these changes in my upcoming PR.